pyomo/util/latex_printer.py

```diff
@@ -0,0 +1,619 @@
+import pyomo.environ as pe
+from pyomo.core.expr.visitor import StreamBasedExpressionVisitor
+from pyomo.core.expr import (
+    NegationExpression,
+    ProductExpression,
+    DivisionExpression,
+    PowExpression,
+    AbsExpression,
+    UnaryFunctionExpression,
+    MonomialTermExpression,
+    LinearExpression,
+    SumExpression,
+    EqualityExpression,
+    InequalityExpression,
+    RangedExpression,
+    Expr_ifExpression,
+    ExternalFunctionExpression,
+)
+
+from pyomo.core.expr.base import ExpressionBase
+from pyomo.core.base.expression import ScalarExpression, _GeneralExpressionData
+from pyomo.core.base.objective import ScalarObjective, _GeneralObjectiveData
+import pyomo.core.kernel as kernel
+from pyomo.core.expr.template_expr import (
+    GetItemExpression,
+    GetAttrExpression,
+    TemplateSumExpression,
+    IndexTemplate,
+)
+from pyomo.core.expr.template_expr import Numeric_GetItemExpression
+from pyomo.core.expr.template_expr import templatize_constraint, resolve_template
```

Comment:
These (plus L#34) should all be in the same import statement

Reply:
Fixed